Magento_Checkout/styles/module/checkout/_tooltip.scss

```diff
@@ -133,7 +133,7 @@ $checkout-tooltip-content-mobile__top           : 30px + $checkout-tooltip-icon-
 @include max-screen($checkout-tooltip-breakpoint__screen-m) {
     .field-tooltip {
         .field-tooltip-content {
-            @extend .abs-checkout-tooltip-content-position-top-mobile;
+            @include abs-checkout-tooltip-content-position-top();
```

Comment:
An alternative to directly include the mixin here would be to use `$screen__m + 1` instead of `$checkout-tooltip-breakpoint__screen-m` as it is used here: https://github.com/SnowdogApps/magento2-theme-blank-sass/blob/master/styles/blocks/_extends.scss#L1132-L1136

Then the `@extend` could persist. But then the media query value would not longer inherit from `$modal-popup-breakpoint-screen__m`.